index.html

```diff
@@ -224,8 +224,20 @@ <h2 id="faq">Frequently Asked Questions <a href="#faq"><svg width="24" height="2
         <p class="font-bold" id="regular-user">
           I'm a regular Terraform user, and I'm not competing with HashiCorp. Why should I care? <a href="#regular-user"><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M13.5442 10.4558C11.8385 8.75022 9.07316 8.75022 7.36753 10.4558L4.27922 13.5442C2.57359 15.2498 2.57359 18.0152 4.27922 19.7208C5.98485 21.4264 8.75021 21.4264 10.4558 19.7208L12 18.1766" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/> <path d="M10.4558 13.5442C12.1614 15.2498 14.9268 15.2498 16.6324 13.5442L19.7207 10.4558C21.4264 8.75021 21.4264 5.98485 19.7207 4.27922C18.0151 2.57359 15.2497 2.57359 13.5441 4.27922L12 5.82338" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/> </svg></a>
         </p>
+
+        <p>
+          There are three reasons that you, as a regular Terraform user, should
+          care.
+        </p>
+        <p>
+          First, if you want to automate your Terraform process, like you would
+          with any other code, you are limited to either spending your precious
+          engineering time and money to build your own solution or using
+          Terraform Cloud. HashiCorp is removing options from you in how best to
```

Comment:
I feel like this is missing a logical step to help readers connect the dots: _why_ does this license change limit that to the only options? I think you'd have to explain that HashiCorp has explicitly said that any CI / CD solution for TF other than TFC/TFE could be a considered a competitor, so either you build your own, or pay them for theirs. Anything else on the market likely has to disappear. 

Also, is that the top reason we think readers should care?

Reply:
>Anything else on the market likely has to disappear.<

Including, potentially, existing/upcoming free open-source solutions so if you build your own, there is the added caveat that you may not be able to open-source it and try to make it a solved problem for the industry.

Every single company/organization will have to either sign-up with Hashicorp or solve the problem again on their own.

The focus is often placed on commercial competitors, but lets not forget that the free open-source ecosystem can itself become a competitor here.